jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile

```diff
@@ -7,6 +7,7 @@ rollingUpgradePipeline(
     params: params,
 
     backend: 'gce',
+    // Version 3.1 is added to cover the issue #5621: upgrade from 3.1 to 3.2 fails
```

Comment:
This comment is irrelevant, let's remove it.

Reply:
Done